tensorboard/plugins/graph/graphs_plugin.py

```diff
@@ -77,7 +78,8 @@ def is_active(self):
         if self._data_provider:
             return False  # `list_plugins` as called by TB core suffices
 
-        return bool(self.info_impl())
+        empty_context = context.RequestContext()  # not used
```

Comment:
No change needed, but honestly it seems simpler just to pass in `plugin_util.context(request.environ)` anyway even if it's not needed by `info_impl()` for this code path rather than to reason through why it's safe to omit it?

Reply:
We don’t have an `environ` because we don’t have a `request`. The bit of
`backend/application.py` that calls this method does, but we don’t have
access to it (and can’t easily get it without breaking `TBPlugin`
compatibility).

Reply:
Oops... duh :/ Sorry!